### PR TITLE
chore(atomic): add publish official script

### PR DESCRIPTION
**Pull request by olamothe**
https://coveord.atlassian.net/browse/KIT-1374

### CODE REVIEW

**Review by ThibodeauJF — APPROVED** [review]

File: packages/atomic-react/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "clean": "rimraf -rf dist",
     "compile": "npm run tsc",
     "tsc": "tsc -p .",
+    "npm:publish": "node ../../scripts/deploy/publish.js",
```
Comment (btaillon) [inline packages/atomic-react/package.json:17]: don't forget to add `atomic-react` to `update-npm-tag.js`

Comment (btaillon) [inline packages/atomic-react/package.json:17]: This publishes as `latest`, is this the intended effect? If you want to publish as alpha, you can make the below change and also add `atomic-react` to `update-npm-tag.js`:
```suggestion
    "npm:publish": "node ../../scripts/deploy/publish.js alpha",
```

Comment (samisayegh) [reply packages/atomic-react/package.json:17]: Indeed, though I do see the [package on npm](https://www.npmjs.com/package/@coveo/atomic-react) tagged as alpha? I wonder how that's happening,

Also, I think you'll need to update the array in the `update-npm-tag.js` script to change the tag to latest,

**Review by samisayegh — APPROVED** [review]

File: packages/atomic-react/package.json (hunk 1 above)
Comment (olamothe) [reply packages/atomic-react/package.json:17]: There's the alpha script just below.

Here, I want to have the release "official", or latest.

Comment (samisayegh) [reply packages/atomic-react/package.json:17]: Why not promote the `alpha` version to `latest`? I think if you publish a version a second time, npm will error and say it's already published.

Comment (olamothe) [reply packages/atomic-react/package.json:17]: Isn't this the exact same mechanism as all other packages, or I'm just terribly confused here ?

Comment (samisayegh) [reply packages/atomic-react/package.json:17]: It is not the same mechanism (please correct me if wrong). I believe publishing directly as latest is an artifact from the early release process that used multiple versions to move a package from dev -> prd (prerelease -> graduate).

The current mechanism publishes as alpha, and then promotes to `beta` in qa and to `latest` in prd.

Comment (olamothe) [reply packages/atomic-react/package.json:17]: Alright, I'll remove the publish script and just add the tag in `update-npm-tag.ts`

**Pull request merged**